cmd/redirector/main.go

```diff
@@ -224,7 +224,7 @@ func main() {
 	if _, err := maxprocs.Set(maxprocs.Logger(func(format string, v ...interface{}) {
 		logger.Info(fmt.Sprintf(format, v...))
 	})); err != nil {
-		logger.Warn("Setting GOMAXPROCS failed: %s", err)
+		logger.Warn("Setting GOMAXPROCS failed: ", "error", err)
```

Comment:
Thank you for fixing the structure to make it more convenient.
In this case, bellow is more desirable.
```suggestion
		logger.Warn("Setting GOMAXPROCS failed", "error", err)
```